src/main/java/com/google/longrunning/OperationsSettings.java

```diff
@@ -66,6 +66,9 @@
   /** The default port of the service. */
   private static final int DEFAULT_SERVICE_PORT = 443;
 
+  private static final String DEFAULT_GENERATOR_NAME = "gapic";
+  private static final String DEFAULT_GENERATOR_VERSION = "0.0.5";
```

Comment:
@garrettjonesgoogle I discussed this earlier today with @shinfan, and my plan is to introduce a new key (`gl-tk`) to indicate the code generator (toolkit) version. The `gapic` token will be used to indicate the generated code version. For languages like Java and C#, the `gccl` and `gapic` versions will always be the same, and they'll be different for others like Python.

You're correct that the presence or absence of the `gccl` and `gapic` keys are important information that we'll track (even if the version is redundant), so that we'll know the relative usage of each level of our client libraries.